frontend/packages/dev-console/src/components/topology/TopologyDataController.tsx

```diff
@@ -116,9 +115,9 @@ export const TopologyDataController: React.FC<TopologyDataControllerProps> = ({
   resourceList,
   serviceBinding,
 }) => {
-  const { resources, utils } = getResourceList(namespace, resourceList);
+  const resourcesList = getResourceList(namespace, resourceList);
```

Comment:
Do we need the changes in this file now can have it was

Reply:
fixed!